scheduler/src/cook/components.clj

```diff
@@ -328,6 +332,22 @@
                           :or {user-limit-per-m 600}} rate-limit]
                      {:user-limit (->UserRateLimit :user-limit user-limit-per-m (t/minutes 1))}))
      :sim-agent-path (fnk [] "/usr/bin/sim-agent")
+     :executor (fnk [[:config {executor {}}]]
+                 (when (str/blank? (:command executor))
+                   (throw (ex-info "Executor command is missing!" {:executor executor})))
```

Comment:
Seems like we shouldn't require the `:executor` config to be present, for two reasons:

1. It breaks backwards compatibility
1. It couples deployment of this change to deployment of the executor on all agents in the Mesos cluster(s)